warehouse/query-core/src/main/java/datawave/query/jexl/lookups/BoundedRangeIndexLookup.java

```diff
@@ -122,7 +122,7 @@ public synchronized void submit() {
                 range = new Range(startKey, true, endKey, literalRange.isUpperInclusive());
             } catch (IllegalArgumentException e) {
                 QueryException qe = new QueryException(DatawaveErrorCode.RANGE_CREATE_ERROR, e, MessageFormat.format("{0}", this.literalRange));
-                log.debug(qe);
+                log.debug(String.valueOf(qe));
```

Comment:
Same thing here. We need the exception treated like a Throwable in order to see the stacktrace. Let's try `log.debug("", qe);`